src/common.h

```diff
@@ -325,7 +325,5 @@ static inline int msg_sendto(int fd, uint8_t *buf, int nbytes,
 
     return ret;
 }
-
-int sock_set_rate_limit(int fd, uint32_t rate_limit);
-
+int sock_set_rate_limit(int fd, uint32_t rate_limit, uint32_t burst_packet_size, uint16_t pkt_size);
```

Comment:
please consider default values for `burst_packet_size` and `pkt_size` to avoid unused variable issue.